ImperatorToCK3/Source/Imperator/Provinces/Province.h

```diff
@@ -30,8 +30,8 @@ namespace ImperatorWorld
 		std::string name;
 		std::string culture;
 		std::string religion;
-		int owner;
-		int controller;
+		int owner = 0;
```

Comment:
ints initialize at 0. No need to set them.

Reply:
I guess that's right, but ReSharper complained about uninitialised member in a class, and it doesn't hurt to satisfy it in this case 😄